lib/classes/events.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_calendar_carousel/classes/event.dart';
+
+class Events {
```

Comment:
I think EventList would be a better naming.

Reply:
Noted, thats sound better. I'll updated it soon